app/_locales/en/messages.json

```diff
@@ -775,7 +775,7 @@
     "message": "Import Wallet"
   },
   "importYourExisting": {
-    "message": "Import your existing wallet using a 12 word seed phrase"
+    "message": "Import your existing wallet using a 24 word seed phrase"
```

Comment:
@diracdeltas thanks for updating those strings. 
Could you make these changes so they happen dynamically at build time in `brave/gulpfile.js/brave-locales.js` instead? Thanks!
We still have I believe no patching to direct files yet, or extremely limited if so.

Reply:
@bbondy done; couldn't test this out since my build is still broken but i grepped the strings in `dist/brave/_locales` and they looked fine